tests/e2e-pw/utils/user-state.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import { PlaywrightWorkerArgs } from '@playwright/test/types/test';
+
+/**
+ * Gets a page which is logged in with an admin session. This is what a merchant will see.
+ */
+export const getMerchantPage = async (
+	browser: PlaywrightWorkerArgs[ 'browser' ]
+) => {
+	const context = await browser.newContext( {
+		storageState: process.env.ADMINSTATE,
+	} );
+	return await context.newPage();
+};
+
+/**
+ * Gets a page logged in with a user session. This is a shopper who has logged into their account.
+ */
+export const getShopperPage = async (
+	browser: PlaywrightWorkerArgs[ 'browser' ]
+) => {
+	const context = await browser.newContext( {
+		storageState: process.env.CUSTOMERSTATE,
+	} );
+	return await context.newPage();
+};
+
+/**
+ * Gets a page not logged in with any session. This will be a shopper who has not logged into their account yet.
+ */
+export const getLoggedOutShopperPage = async (
+	browser: PlaywrightWorkerArgs[ 'browser' ]
+) => {
+	const context = await browser.newContext( {
+		storageState: null,
+	} );
+	return await context.newPage();
+};
```

Comment:
Nice solution, @opr! 👏 Looking at https://playwright.dev/docs/api/class-browsercontext, I see that Playwright provides a solution for disposing the context when it's no longer needed. This is the corresponding code:
```tsx
await context.close();
```

I don't think it's worth creating a utility function for that. Would you suggest adding that code once the context is no longer required? Something along this line:

```tsx
const page = getMerchantPage();
// ... do some magic
await context.close();
```